crates/vm2/src/tracing.rs

```diff
@@ -458,7 +454,7 @@ mod test {
             1000,
             crate::Settings {
                 default_aa_code_hash: [0; 32],
-                evm_interpreter_code_hash: [0; 32],
```

Comment:
This pervasive renaming seems somewhat unnecessary. The other renamings did improve clarity.